### PR TITLE
Prep Vendor quality of life change.

### PR DESCRIPTION
**Pull request by Huffie56**
## About The Pull Request

I open an issue https://github.com/cmss13-devs/cmss13/issues/1449
I am looking what could be improve into all the prep vendors to make them easier to use and more coherent.  
Still in work do not merge

## Why It's Good For The Game

1-having ammo bellow the corresponding gun will decrease time to equip and make it easier for new player to equip.
2-Having a consistent order of the equipment across all the vendors will make it easier to know where to look.

## Where to find the codes involve in the different vendors

1-ColMarTech Surplus Uniform Vendor https://github.com/cmss13-devs/cmss13/blob/master/code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm line 231.
2-ColMarTech Automated Weapons Rack https://github.com/cmss13-devs/cmss13/blob/master/code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm line 1

## Changelog

:cl:
code: Changing so that Below every weapon the corresponding ammo is available in the ColMarTech Automated Weapons Rack.
code: 2-Changing in ColMarTech Surplus Uniform Vendor so that the orders of basic items correspond to the ColMarTech Automated.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
Anyone know why ColMarTech Surplus Uniform Vendor have multiple version one for each squad when at this moment they are accessible by every squad sharing the same prep?
https://github.com/cmss13-devs/cmss13/blob/master/code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm line 23.
Feel like the code for the vendor specific to squads aren't being use and are legacy??

**Comment by Huffie56** [comment]
maybe i am going crazy but could it be possible to have one vendor that load the items a marines as access to via a list instead of having X different vendors? would it be easier to maintain and modify?

File: code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm [hunk 1]
```diff
@@ -7,33 +7,29 @@
 	req_access = list()
 	req_one_access = list(ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_PREP, ACCESS_MARINE_CARGO)
 	hackable = TRUE
-
+//changing it so that the gun as the corresponding ammo bellow it
```
Comment (fira) [inline code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm:10]: don't add "historical" comments, that's what git's for

File: code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm [hunk 2]
```diff
@@ -229,7 +225,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_marine, list(
 	headset_type = /obj/item/device/radio/headset/almayer/marine/delta
 
 //------------SQUAD PREP UNIFORM VENDOR---------------
-
+//changing how the order of the items in the list to make it fit the ColMarTech Automated Marine Equipment Rack.
```
Comment (fira) [inline code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm:228]: same

**Review by fira — COMMENTED** [review]
that aside, makes sense to me

**Comment by Huffie56** [comment]
No real point pushing this since TGUI rework will change all the code around vendors.....

**Pull request closed without merging**